src/main/helm/helm-release-manager.ts

```diff
@@ -191,7 +191,6 @@ async function getResources(name: string, namespace: string, kubeconfigPath: str
   ];
   const kubectlArgs = [
     "get",
-    "--namespace", namespace,
```

Comment:
Why would removing the namespace mean that the "correct" resources are retrieved?

Reply:
Other changes fixes https://github.com/lensapp/lens/issues/5849.

But more changes are coming when I get the behaviours in because I can already see two other blindspots.